code/modules/projectiles/guns/flamer.dm

```diff
@@ -359,12 +436,15 @@
 		/obj/item/attachable/hydro_cannon,
 	)
 	var/last_use
+	var/hydro_active
+	var/water_count
```

Comment:
Is this a bool and a integer?
If yes please define as such
(var/bool = FALSE, var/integ = 0)